### PR TITLE
use Object instead of dynamic to workaround analyzer const bug

### PR DESCRIPTION
**Pull request by jmesserly**
R= @jakemac53

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: smoke
-version: 0.3.1+1
+version: 0.3.2
```
Comment (jakemac53) [inline pubspec.yaml:2]: I think this could be a +2 since its really just a bug fix?

**Comment by jakemac53** [comment]
update changelog too, otherwise lgtm

**Comment by jmesserly** [comment]
ah good call

**Comment by jmesserly** [comment]
Thanks!

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+#### 0.3.2
+  * Work around an issue running Dart analyzer on the generated code, if the
+    `dynamic` type appeared in the output. Smoke will now use `Object` instead.
```
Comment (sigmundch) [inline CHANGELOG.md:3]: Just to clarify, I suppose we plan to revert this once the analyzer fix lands, right?

Comment (jakemac53) [inline CHANGELOG.md:3]: Yes

**Pull request merged**